src/components/dashboard/FeedItems/EventStyles.js

```diff
@@ -67,9 +89,10 @@ export const listStyles = StyleSheet.create({
     alignItems: 'center'
   },
   date: {
-    fontSize: normalize(8),
+    fontSize: normalize(10),
     color: 'rgba(75, 75, 75, 0.8)',
-    marginLeft: 'auto'
+    marginLeft: 'auto',
+    fontFamily: 'Roboto-Regular'
```

Comment:
We should need to chose fontfamily on each component. Or at least we should be picking from the theme

Reply:
Done